package.json

```diff
@@ -42,8 +42,10 @@
     "semver": "^5.1.0"
   },
   "devDependencies": {
-    "typescript": "^1.9.0-dev.20160418",
-    "typings": "^0.6.8",
+    "@types/progress": "^1.1.8",
```

Comment:
Also I should mention I did actually `node run lib/tattoo.js` after build to verify this works.

Runs fine in node 4.  I have build issues with nodegit and node 6 due to openssl or something so haven't verified node 6 is happy yet.